src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,63 @@
+import { CustomTodo } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+import TempTodo from '../TempTodo/TempTodo';
+import TodoComponent from '../Todo/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[];
+  customTodo: CustomTodo | null;
+  onDeleteTodo: (todoId: number) => void;
+  deletingCompleted: boolean;
+  onToggleStatus: (todo: Todo) => void;
+  isChanging: boolean;
+  isToggling: boolean;
+  onTitleChange: (todo: Todo, value: string) => void;
+};
+
+const TodoList: React.FC<Props> = ({
+  todos,
+  customTodo,
+  onDeleteTodo,
+  deletingCompleted,
+  onToggleStatus,
+  isChanging,
+  isToggling,
+  onTitleChange,
+}) => {
+  let currentlyCanging = Filter.NONE;
+
+  if (isToggling) {
+    if (todos.some(todo => !todo.completed)) {
+      currentlyCanging = Filter.ACTIVE;
+    } else {
+      currentlyCanging = Filter.COMPLETED;
+    }
+  }
```

Comment:
Create util for it and use `currentlyChanging={getCurrentlyCanging()}`

Reply:
i prefere it like this